styles/app.scss

```diff
@@ -1,6 +1,124 @@
 // @import 'yoga';
 // @import 'normalize';
 
-// $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
-// @import 'font-awesome';
+$fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
+@import 'font-awesome';
 @import 'box-sizing';
+@import 'variables';
+
+body {
+  margin: 0;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+figure {
+  margin-left: 0;
+}
+
+.app {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+
+  &__nav {
+    padding: 0 $gutter;
+    background: $brand-gross-orange;
+    color: $brand-white;
+    font-family: sans-serif;
+  }
+
+  &__main {
+    flex-grow: 1;
+    padding: 0 $gutter;
+  }
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  max-width: 60rem;
+  margin: auto;
+}
+
+.title {
+  flex-grow: 1;
+  padding: 0;
+}
+
+.fa-plus-circle {
+  color: $brand-white;
+  font-size: 1.5rem;
+}
+
+.nav-form {
+  display: flex;
+  width: 100%;
+  padding: 0 $gutter;
+  background: $brand-dark-gray;
+  color: $brand-white;
+  font-family: sans-serif;
+}
+
+.form-list {
+  width: 60rem;
+}
+
+.puppy-card {
+  display: flex;
+  width: 60rem;
+  background: $brand-light-gray;
+  color: $brand-black;
+  font-family: sans-serif;
+}
+
+.add-puppy {
+  background: $brand-dark-gray;
+}
+
+.card-info {
+  flex-grow: 1;
+}
+
+.button-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.button {
+  background: $brand-gross-orange;
+  color: $brand-white;
+}
+
+.puppies {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  background: $brand-gross-orange;
+  color: $brand-white;
+}
+
+.dropdown-button {
+  background: $brand-gross-orange;
+  color: $brand-white;
+}
+
+.top-nav__form {
+  color: $brand-white;
+  padding:0 10rem;
```

Comment:
This should have the same padding as `.app__nav`

Then wrap the form in a `container` div to get the max-width effect

![screen shot 2016-07-17 at 8 02 08 pm](https://cloud.githubusercontent.com/assets/2532004/16904329/5e78db9c-4c59-11e6-90b6-ad674a843bd8.png)